test/system-test/utils.py

```diff
@@ -173,10 +177,42 @@ def vote(proposal_id, vote, get_logs=False):
 
-
-def get_test_wrapping_key():
-    return '"-----BEGIN PUBLIC KEY-----\\nMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA0L9FDBjydkdstv7OKqkw\\ndMiugRqlSHC9Lchfd7jh5uCzv602LhlBJQeEFYchvaEquISLQFoZxEkpGEbEb15v\\nN2dKwTCi0ioEGBidtFmuKiVZqf46Hbnw4OdinQHrlGO2PRsRE+DYPOy6xrZTKEnD\\nK+OnHDsZ0U2qNJ80IjbxcC83lQpaMx8Ij8AddGY9Msv0TMMgaVsrDaQYLC8tmJih\\nxVI2f5BHFbTy3pw4Sq9xp6se+S/ycOUF0M6RFArZcD9uR/NxJKFJJlGOKskgwLnT\\nlhPck4sGeLhHUydRdqw0+h8EDsTIUysNxMOYtZETaVuBS+ISMy8+WQgVPs12Ujr3\\n17kaHeZr8Lq0bwaFHruRBpNwqtUCBv57IBpe7hnEDDdvOvN/tPubf1dv3HxEt42T\\nqqfozS+a9+1hcI8hpNlEjh+qcy1BmhSOXvmRzlhauX4xv6OLCNkRxo6x2Q/1moDC\\nPgnaJJVIuESr07xnC8fk43i5qFzEXQO3hwNsjd7sqFBBTb6t5N6Nm37mMNTqsDky\\nniAeFG+1gK/UD+cMPfbUIDaCqpCDwrTX0gMqqUTDG6eNPmQaOa+slici3h9WLaNy\\nmEzfKqJMBggKib/+e4Eb/ENdvxeT1X2YXpZ3tjZE+bRoiDgN4FYqBzYtZ/ieRcsq\\n4fPqgZPbh+ivT2o7QutzWH0CAwEAAQ==\\n-----END PUBLIC KEY-----\\n"'  # pragma: allowlist secret
-
+def get_test_public_wrapping_key():
+    with open(os.path.join(os.path.dirname(__file__), '../data-samples/publicWrapKey.pem'), 'r') as file:
+        key = file.read()
+        # Escape newlines and wrap the key in double quotes
+        key = key.replace("\n", "\\n")
+        #return f"{key}"
+        return f'"{key}"'
+
+def get_test_private_wrapping_key():
+    with open(os.path.join(os.path.dirname(__file__), '../data-samples/privateWrapKey.pem'), 'r') as file:
+        key = file.read().strip()
+        return key
+"""
```

Comment:
Why is this multi-line string here?

Reply:
My original attempt to decrypt. It is still in comments. I will remove it later.

Reply:
It's not a comment, it's a multiline string literal